src/description.cpp

```diff
@@ -1086,8 +1086,13 @@ Description::Audio::Audio(string mid, Direction dir)
     : Media("audio 9 UDP/TLS/RTP/SAVPF", std::move(mid), dir) {}
 
 void Description::Audio::addAudioCodec(int payloadType, string codec, optional<string> profile) {
-	if (codec.find('/') == string::npos)
-		codec += "/48000/2";
+	if (codec.find('/') == string::npos){
+        if(0 == codec.compare("OPUS"))
+            codec += "/48000/2";
+        else if(0 == codec.compare("PCMA") || 0 == codec.compare("PCMU"))
+            codec += "/8000/1";
```

Comment:
You could check for "PCMA" or "PCMU" or fallback to default to keep the former behavior. Also, calling `compare` seems convoluted here:
```suggestion
        if(codec == "PCMA") || codec == "PCMU")
            codec += "/8000/1";
        else
            codec += "/48000/2";
```